### DISCUSSION TITLE
Reads dropped due to the lack of a clone sequence

### DISCUSSION THREAD

**Discussion by zznx** [milaboratory/mixcr#1157] [Q&A]
Hi,
What causes this? 
Is there any specific code for that? And how do we get this sequence?
`Reads dropped due to the lack of a clone sequence, percent of total: 215992 (39.52%)`

Thanks
## MiXCR report files 

Command line arguments: assemble --write-alignments -t 16 -ObadQualityThreshold=10 -f --report PC-FR2.IonXpress_061_assemble.report PC-FR2.IonXpress_061_align.vdjca PC-FR2.IonXpress_061_assemble.clna
Analysis time: 49.06s
Final clonotype count: 12029
Average number of reads per clonotype: 7.37
Reads used in clonotypes, percent of total: 88653 (16.22%)
Reads used in clonotypes before clustering, percent of total: 92255 (16.88%)
Number of reads used as a core, percent of used: 92052 (99.78%)
Mapped low quality reads, percent of used: 203 (0.22%)
Reads clustered in PCR error correction, percent of used: 3602 (3.9%)
Reads pre-clustered due to the similar VJC-lists, percent of used: 0 (0%)
Reads dropped due to the lack of a clone sequence, percent of total: 215992 (39.52%)
Reads dropped due to low quality, percent of total: 0 (0%)
Reads dropped due to failed mapping, percent of total: 26478 (4.85%)
Reads dropped with low quality clones, percent of total: 0 (0%)
Clonotypes eliminated by PCR error correction: 1323
Clonotypes dropped as low quality: 0
Clonotypes pre-clustered due to the similar VJC-lists: 0
IGH chains: 12028 (99.99%)
IGL chains: 1 (0.01%)

**Comment by mizraelson** [comment]
Hi, this means that the reads don't cover the assembling feature (for instance if you are assembling clones including CDR2 but some reads don't cover it). Can you share the set of commands you used for this data?

**Comment by zznx** [comment]
Can parameters improve this result?

`Command line arguments: align -s human -t 16 -f -OminSumScore=200 --library imgt.201946-3.sv6.json -OsaveOriginalReads=true --report PC-FR2.IonXpress_061_align.report PC-FR2.IonXpress_061.fastq PC-FR2.IonXpress_061_align.vdjca`

`Command line arguments: assemble --write-alignments -t 16 -ObadQualityThreshold=10 -f --report PC-FR2.IonXpress_061_assemble.report PC-FR2.IonXpress_061_align.vdjca PC-FR2.IonXpress_061_assemble.clna`

**Comment by mizraelson** [comment]
What type of data do you have? Is it a targeted TCR/BCR library? Can you tell more about its structure? Generally this means that some portion of your reads doesn't cover the assembling feature, which is 'CDR3' by default.

**Comment by zznx** [comment]
Does this help you?
```
>>> Read ids: 0


                                                                                                    
                                                                           FR2><CDR2           C    
                 D  V  K  R  R  P  V  V  R  Q  A  P  G  K  G  L  E  W  V  S  V  I  Y  S  C  G  S    
    Quality     54555755245445355666666656656656564666566666465544666656666666566656666666554566    
    Target0   0 GATGTAAAACGACGGCCAGTGGTCCGCCAGGCTCCAGGGAAGGGGCTGGAGTGGGTCTCAGTTATTTATAGCTGTGGTAG 79   Score
IGHV3-66*03 107                     ggtccgccaggctccagggaaggggctggagtgggtctcagttatttatagctgtggtag 166  930
IGHV3-66*02 107                     ggtccgccaggctccagggaaggggctggagtgggtctcagttatttatagcGgtggtag 166  906
IGHV3-53*02 107                     ggtccgccaggctccagggaaggggctggagtgggtctcagttatttatagcGgtggtag 166  888
IGHV3-53*04 107                     ggtccgccaggctccagggaaggggctggagtgggtctcagttatttatagcGgtggtag 166  888
IGHV3-66*01 107                     ggtccgccaggctccagggaaggggctggagtgggtctcagttatttatagcGgtggtag 166  888
        0R0   0 gatgtaaaacgacggccagtggtccgccaggctccagggaaggggctggagtgggtctcagttatttatagctgtggtag 79  

                                                                                                    
                DR2><FR3                                                                            
                  T  Y  Y  A  D  S  V  K  G  R  F  T  I  S  R  D  N  S  K  N  T  L  Y  L  Q  M      
    Quality     66666666666666666666666666666666666666455656666666666666566566655666666534464666    
    Target0  80 CACATACTACGCAGACTCCGTGAAGGGCCGATTCACCATCTCCAGAGACAATTCCAAGAACACGCTGTATCTTCAAATGA 159  Score
IGHV3-66*03 167 cacatactacgcagactccgtgaagggccgattcaccatctccagagacaattccaagaacacgctgtatcttcaaatga 246  930
IGHV3-66*02 167 cacatactacgcagactccgtgaagggccgattcaccatctccagagacaattccaagaacacgctgtatcttcaaatga 246  906
IGHV3-53*02 167 cacatactacgcagactccgtgaagggccgattcaccatctccagagacaattccaagaacacgctgtatcttcaaatga 246  888
IGHV3-53*04 167 cacatactacgcagactccgtgaagggccgattcaccatctccagaCacaattccaagaacacgctgtatcttcaaatga 246  888
IGHV3-66*01 167 cacatactacgcagactccgtgaagggcAgattcaccatctccagagacaattccaagaacacgctgtatcttcaaatga 246  888
        0R0  80 cacatactacgcagactccgtgaagggccgattcaccatctccagagacaattccaagaacacgctgtatcttcaaatga 159 

                                                                                          DP>       
                                               FR3><CDR3    V>   <DPDP><D                 D><DP     
                N  S  L  R  A  E  D  T  A  V  Y  Y  C  A  R  E  C  T  P  G  I  A  V  A  G  T  P     
    Quality     66666566666666666666666665556655556666655555555555566656745566666666566656666665    
    Target0 160 ACAGCCTGAGAGCTGAGGACACGGCTGTGTATTACTGTGCGAGAGAGTGTACTCCGGGTATAGCAGTGGCTGGTACGCCC 239  Score
IGHV3-66*03 247 acagcctgagagctgaggacacggctgtgtattactgtgcgagaga                                   292  930
IGHV3-66*02 247 acagcctgagagctgaggacacggctgtgtattactgtgcgaga                                     290  906
IGHV3-53*02 247 acagcctgagagcCgaggacacggcCgtgtattactgtgcgagaga                                   292  888
IGHV3-53*04 247 acagcctgagagctgaggacacggcCgtgtattactgtgcgagaga                                   292  888
IGHV3-66*01 247 acagcctgagagcCgaggacacggctgtgtattactgtgcgagaga                                   292  888
IGHD6-19*01  16                                                  tac-ccgggtatagcagtggctggtacg    42   123
        0R0 160 acagcctgagagctgaggacacggctgtgtattactgtgcgagagagtgtactccgggtatagcagtggctggtacgccc 239 

                                                  
                           <J                     
              *  K  I  L  P  Y   S  V  W  T _     
 Quality     6666661555555535555 5545554342222    
 Target0 240 TAAAAAATCCTCCCCTACT-CGGTATGGACGTC 271  Score
IGHJ6*01  30               ctactAcggtatggacgtc 48   78
IGHJ6*02  30               ctactAcggtatggacgtc 48   78
IGHJ6*04  30               ctactAcggtatggacgtc 48   78
     0R0 240 taaaaaatcctcccctact-cggtatggacgtc 271 

Filtered: 1 / 334725 = 2.9875270744641124E-4%
```

**Comment by mizraelson** [comment]
Is it a set of J multiplex primers? from here I see that the last three nucleotides that belong to CDR3 are not present in the sequence (tgg), that's why CDR3 is not annotated here. Can you share a pair of raw reads (forward and reverse) preferably the one with IGHJ6 in it if its a multiplex?

**Comment by zznx** [comment]
Yes
`Is it a set of J multiplex primers?`
And I don't have any other data
